src/forms/templates/utils/index.js

```diff
@@ -1,9 +1,10 @@
 const _checkIfHidden = (name, uiSchema) => {
   return (
-    uiSchema &&
-    uiSchema[name] &&
-    uiSchema[name]["ui:options"] &&
-    uiSchema[name]["ui:options"].hidden
+    (uiSchema &&
+      uiSchema[name] &&
```

Comment:
Can we remove the `uiSchema[name]` branch? It seems to be unused, do we need it in CAP?